### DISCUSSION TITLE
Tablesort source update: CDNJS to UNPKG

### DISCUSSION THREAD

**Discussion by menelli84** [squidfunk/mkdocs-material#3789] [Ideas]
Hello Martin! 

First, thanks for your Material for Mkdocs project, which has helped me a lot! I am very grateful for it. 

A constructive suggestion I found, is relative to the tablesort function, where in your documentation, you indicate the CDNJS repository (https://cdnjs.cloudflare.com/ajax/libs/tablesort/5.3.0/tablesort.min.js), however when trying to access it, I saw that it is showing ERROR 404. 

When searching another repository of this javascript library, I found UNPKG, where I believe it is useful to return such a function to your beautiful documentation site. 

The link is this: https://unpkg.com/tablesort@5.3.0/src/tablesort.js

I took the liberty of suggesting this to you, because I really like your work and the fact that you have a large part of it for free is commendable. 

Finally, know that as soon as I get clients using MKDOCS+Material for MKDOCS, I will sign up for the **Insiders** program, to have access to deeper customizations. 

Thanks again. 

Hugs from Brazil - Renan Menelli.

**Comment by squidfunk** [comment]
Thanks! I think it's a good idea to switch to unpkg.com, as it's definitely the better CDN and we already use it for Mermaid.js  – I went ahead and changed it in b11a4741c. Note that I used the minified version, as it's smaller. Also thanks for considering sponsoring the project, it would help a lot! 😊
